google/auth/_default_async.py

```diff
@@ -0,0 +1,333 @@
+# Copyright 2015 Google Inc.
```

Comment:
Please use git mv when moving files-- otherwise it's hard to see what was changed in addition to the file being renamed.

Reply:
Ok, noted, will do in the next round